assets/css/_vehicle_properties_panel.scss

```diff
@@ -0,0 +1,70 @@
+.m-vehicle-properties-panel {
+  background-color: $pale-grey;
+  height: 100%;
+  max-width: 100%;
+  position: fixed;
+  right: 0;
+  top: inherit;
+  width: 23.5rem;
+  z-index: 1001;
+
+  hr {
```

Comment:
Should the `hr` get its own class to make selecting it easier? (Same for `dl`, `dt`, and `dd` below which also have tag selectors.)

Reply:
Yeah, I always wonder when then gets more annoying than it's worth, but sure, can do.